action.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Telemetry Gathering'
+description: 'Extract all relevant JIRA tickets from a given GitHub event payload and write those values to a user-specified webhook endpoint.'
+inputs:
+  collector-endpoint:
+    description: "A webhook to receive the Jira ticket-containing payload"
+    required: true
+  collector-token:
+    description: "Token used to authenticate GitHub Action to the webhook endpoint"
+    required: true
+  jira-keys:
+    description: "Prefixes to search for in Jira tickets"
+    required: true
+  event:
+    description: "GitHub API payload for this event"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: "Setup Python"
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.x"
+    - name: "Install Necessary Package"
+      run: |
+        python -m pip install --upgrade pip
+        pip install requests
+    - name: "Send Payload"
+      shell: bash
+      run: python ./github/telemetry-gathering/send_webhook_payload.py
```

Comment:
Let's just put this in the root of the action. No need for folders. Let's just call it `telemetry.py`